ext/com_dotnet/com_typeinfo.c

```diff
@@ -228,12 +228,12 @@ PHP_COM_DOTNET_API zend_result php_com_import_typelib(ITypeLib *TL, int mode, in
 					ZVAL_LONG(&c.value, Z_LVAL(value));
 					if (mode & CONST_PERSISTENT) {
 						/* duplicate string in a persistent manner */
-						c.name = zend_string_dup(const_name, /* persistent */ true);
+						name = zend_string_dup(const_name, /* persistent */ true);
 						zend_string_release_ex(const_name, /* persistent */ false);
 					} else {
-						c.name = const_name;
+						name = const_name;
 					}
-					zend_register_constant(&c);
+					zend_register_constant(name, &c);
```

Comment:
You probably need to release "name" in case of failure.

Reply:
As far as I saw, I have to release the `name` any way because `zend_hash_add_constant()` (and ultimately `zend_hash_add`) increases the refcount for non-interned strings.